recipes/libyuv/all/conanfile.py

```diff
@@ -48,11 +49,11 @@ def layout(self):
 
     def requirements(self):
         if self.options.with_jpeg == "libjpeg":
-            self.requires("libjpeg/9e")
+            self.requires("libjpeg/9f")
         elif self.options.with_jpeg == "libjpeg-turbo":
-            self.requires("libjpeg-turbo/3.0.1")
+            self.requires("libjpeg-turbo/[>=3.0.3 <4]")
         elif self.options.with_jpeg == "mozjpeg":
-            self.requires("mozjpeg/4.1.3")
+            self.requires("mozjpeg/[~4.1.5]")
```

Comment:
```suggestion
            self.requires("mozjpeg/4.1.5")
```

We do not control the compatibility model of `mozjpeg` so having a fixed version is more secure for users.